cli/src/common_utils/utility_scripts.sh

```diff
@@ -142,7 +146,12 @@ function ensure_healthy_cluster {
 
                 if [[ ${attempt} -ge 20 ]]; then
                     _lerror "Giving up after ${attempt} attempts."
-                    exit 20
+                    if [[ "$_exit_on_err" == true ]]; then
```

Comment:
So we never invoke this function with `_exit_on_err == true`? because in that case `ret_ensure_healthy_cluster` will remain 0.